Client/src/components/common/Link/index.js

```diff
@@ -3,9 +3,9 @@ import { Link } from 'react-router-dom';
 
 import './index.css'
 
-export default function link({ title, to }) {
+export default function SubLink({ title, to }) {
   return (to ?
-    <Link to={to}>
+    <Link  className='sidebar-linklist-link'to={to}>
```

Comment:
```suggestion
    <Link  className='sidebar-linklist-link' to={to}>
```
for readability reasons